.github/workflows/publish-release.yml

```diff
@@ -1,29 +1,110 @@
 name: Publish Release
 
 on:
-  pull_request:
-    types: [closed]
+  workflow_call:
+    secrets:
+      NPM_TOKEN:
```

Comment:
Note that we will need to add an `NPM_TOKEN` in order to merge this pull request.

Reply:
This has been setup